packages/language/src/v1beta1/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var languageServiceApi = require('./language_service_api');
+
+function v1beta1(options) {
```

Comment:
You should be able to remove the `v1beta1` function and just export it later:

```js
module.exports = languageServiceApi;
```

The constants will be picked up as well, which I believe makes this file unnecessary. Maybe `langage_service_api.js` should just be `index.js`?